server/amp/handlers/visual-stories-bookend.js

```diff
@@ -0,0 +1,75 @@
+const get = require("lodash/get");
+
+function getImageUrl(story, config) {
```

Comment:
Rename to something meaningful like `getStoryUrl`

Reply:
Ok